### PR TITLE
prebuilt_docs_updates

### PR DESCRIPTION
**Pull request by akankshat100ms**

### CODE REVIEW

File: docs/get-started/v2/get-started/prebuilt/Screens-and-components.mdx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+Enabling this makes the local peer's *(for the selected role)* tile in a smaller inset window as default, alternatively if it's disabled, the local tile will be part of the grid view. Join with atleast two peers to preview this configuration in action.
+
+#### Prominent roles
+Defining one or more roles as prominent gives them higher tile view prominence for the selected role. For example:
+- For a 1:1 call, define Host as prominent tile for Guest and vice-versa. 
+- For a webinar, set Host as the prominent tile for all other roles. Every role would see Host as the primary tile in the room
+- For a mobile-first livestreaming view, set Broadcaster as prominent tile for every other role, including Broadcaster themselves.
+
+#### Can spotlight peer
+Allow this config for roles who can spotlight others or themselves in the room. Spotlighting a tile reflects for everyone in the room.
```
Comment (saikatmitra91) [inline docs/get-started/v2/get-started/prebuilt/Screens-and-components.mdx:75]: explain how the UI experience of video tile is similar to the prominent role and also explain how this is slightly different

File: docs/get-started/v2/get-started/prebuilt/Screens-and-components.mdx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+### Be Right Back (BRB)
+Depending on your use case, allow roles to set themselves on BRB mode. If disabled, this component will not be visible to the selected role.
+
+
+### Emoji Reactions
+Enabling this for a role allows its peers to send emoji reactions in the room. 
+
+### Bring others on Stage
+This component allows the selected role to allow/deny stage request from off stage participants. This component is available on both conferencing and live-streaming templates. By default, this feature is enabled for Hosts, Broadcasters and Co-broadcasters, where viewer(non-publishing) roles would have a hand-raise option to request stage access. If your use case doesn't require this feature, simply disable this component. Read more on Bring on stage feature.
```
Comment (saikatmitra91) [inline docs/get-started/v2/get-started/prebuilt/Screens-and-components.mdx:90]: lets explain whats on stage and off stage ?

**Review by saikatmitra91 — CHANGES_REQUESTED** [review]

File: docs/get-started/v2/get-started/prebuilt/Appearance.mdx [hunk 3]
```diff
@@ -0,0 +1,98 @@
+---
+title: Appearance
+nav: 2.1
+---
+
+Customize the visual aspects of your Prebuilt app, ensuring it aligns with your brand identity and user experience preferences. Preview your changes on the go, once saved you can test these changes on Prebuilt links and our native apps. This documentation will guide you through the various appearance customization options available within the Prebuilt Customizer.
```
[the remark this answers is not in the document]
Comment (akankshat100ms) [reply docs/get-started/v2/get-started/prebuilt/Appearance.mdx]: updated

File: docs/get-started/v2/get-started/prebuilt/Screens-and-components.mdx [hunk 4]
```diff
@@ -0,0 +1,105 @@
+---
+title: Screens and Components
+nav: 2.2
+---
+
```
[the remark this answers is not in the document]
Comment (akankshat100ms) [reply docs/get-started/v2/get-started/prebuilt/Screens-and-components.mdx:5]: this is conveyed using the video below.

File: docs/get-started/v2/get-started/prebuilt/Appearance.mdx [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+<div style={{textAlign: 'center'}}>
+   
+![LogoCustomisation](/docs/v2/LogoCustomisation.gif)    
+</div>
+
+## Switching Between Dark and Light Themes
+
+Prebuilt offers two distinct themes for your app: a dark theme and a light theme. You can choose the one that best suits your app’s aesthetics or user preferences.
+
+Once you toggle the theme switch to select either the dark or light theme, your Prebuilt app will be displayed with the selected theme.
```
[the remark this answers is not in the document]
Comment (akankshat100ms) [reply docs/get-started/v2/get-started/prebuilt/Appearance.mdx:24]: @chrisb-100ms can you check this?

File: docs/get-started/v2/get-started/prebuilt/Appearance.mdx [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+<div style={{textAlign: 'center'}}>
+   
+![LogoCustomisation](/docs/v2/ColorTokens.gif)    
+</div>
+
+### **Primary Colors**
+
+The primary colors play a crucial role for defining UI elements that reflect the brand for main elements like primary buttons. There are four core primary roles you can customize:
+
+- **Primary Bright**
```
[the remark this answers is not in the document]
Comment (akankshat100ms) [reply docs/get-started/v2/get-started/prebuilt/Appearance.mdx:48]: updated

File: docs/get-started/v2/get-started/prebuilt/Screens-and-components.mdx (hunk 1 above)
Comment (akankshat100ms) [reply docs/get-started/v2/get-started/prebuilt/Screens-and-components.mdx:76]: done

**Comment by KaustubhKumar05** [comment]
@akankshat100ms can you check the vale typo errors? If valid terms are being flagged, they can be added in `accept.txt`and vale will skip those terms. Like codec, tldraw or variable names like keyof :
https://github.com/100mslive/100ms-docs/pull/1841/checks?check_run_id=18431830227
cc @hdz-666

**Review by mntrspace — CHANGES_REQUESTED** [review]

File: docs/get-started/v2/get-started/prebuilt/quickstart.mdx [hunk 7]
```diff
@@ -0,0 +1,186 @@
...
+
+
+## Pre-requisites 
+<StepsContainer id="prerequisites">
+
+### Set up a template
+A [template](/v2/get-started/concepts/templates-and-roles) governs how its Prebuilt UI functions. Create a template using dashboard or create [template via API](/v2/api-reference/policy/create-template-via-api).
+
+### Create a room
+When you create a template on 100ms dashboard, a room gets created by default. You can programatically create rooms using [create room API](/v2/api-reference/Rooms/create-via-api).
```
[the remark this answers is not in the document]
Comment (akankshat100ms) [reply docs/get-started/v2/get-started/prebuilt/quickstart.mdx]: updated

File: docs/get-started/v2/get-started/prebuilt/quickstart.mdx [hunk 8]
```diff
@@ -0,0 +1,186 @@
...
+title: Quickstart
+nav: 2.3
+---
+
+
+## Pre-requisites 
+<StepsContainer id="prerequisites">
+
+### Set up a template
+A [template](/v2/get-started/concepts/templates-and-roles) governs how its Prebuilt UI functions. Create a template using dashboard or create [template via API](/v2/api-reference/policy/create-template-via-api).
```
[the remark this answers is not in the document]
Comment (akankshat100ms) [reply docs/get-started/v2/get-started/prebuilt/quickstart.mdx]: updated

File: docs/get-started/v2/get-started/prebuilt/overview.mdx [hunk 9]
```diff
@@ -1,177 +1,65 @@
...
 
-// add pre-built component
-const ConferencingScreen = () => {
-  return <HMSPrebuilt roomCode="<room-code>" />;
-};
-```
+## Customise Prebuilt 
+100ms Prebuilt abstracts complex nuances around building audio/video use cases; these are built on top of two core entities which help build customized experiences using:
+    - **Templates** - Customize how your rooms function, such as which roles can join with what permissions, features like recording, livestreaming, transcriptions, and more. Read more on [Templates and Roles](/get-started/v2/get-started/concepts/templates-and-roles)
+    - **Prebuilt Customizer** - a no-code customisation tool that allows for quick and powerful adjustments using straightforward enable/disable toggles. With its real-time preview feature, you can conveniently review changes as you make them on the fly. There are two levels of no-code customisation available:
```
[the remark this answers is not in the document]
Comment (akankshat100ms) [reply docs/get-started/v2/get-started/prebuilt/overview.mdx]: done

File: docs/get-started/v2/get-started/prebuilt/overview.mdx [hunk 10]
```diff
@@ -1,177 +1,65 @@
...
-```
-</Tab>
 
-<Tab id='sample-code-1'>
-```dart section=SampleCode tab=Flutter
+## Experience Prebuilt
 
-// import roomkit package
-import 'package:hms_room_kit/hms_room_kit.dart';
+100ms Prebuilt inherits its behavior from its template configuration; using this template as its core, it creates a default Prebuilt UI for each role in the template.  Here, the template defines the roles and the permissions they join a room with, and supports features like recording, livestreaming, transcriptions etc, whereas the UI extends features across different screens and components on 100ms Prebuilt.
```
[the remark this answers is not in the document]
Comment (akankshat100ms) [reply docs/get-started/v2/get-started/prebuilt/overview.mdx:40]: updated

File: docs/get-started/v2/get-started/prebuilt/Appearance.mdx (hunk 5 above)
[the remark this answers is not in the document]
Comment (akankshat100ms) [reply docs/get-started/v2/get-started/prebuilt/Appearance.mdx:24]: updated

**Comment by akankshat100ms** [comment]
> @akankshat100ms can you check the vale typo errors? If valid terms are being flagged, they can be added in `accept.txt`and vale will skip those terms. Like codec, tldraw or variable names/keywords like keyof : https://github.com/100mslive/100ms-docs/pull/1841/checks?check_run_id=18431830227 cc @hdz-666

this is fixed, thanks!

**Comment by akankshat100ms** [comment]
> @akankshat100ms can you check the vale typo errors? If valid terms are being flagged, they can be added in `accept.txt`and vale will skip those terms. Like codec, tldraw or variable names/keywords like keyof : https://github.com/100mslive/100ms-docs/pull/1841/checks?check_run_id=18431830227 cc @hdz-666

**Comment by hdz-666** [comment]
> > @akankshat100ms can you check the vale typo errors? If valid terms are being flagged, they can be added in `accept.txt`and vale will skip those terms. Like codec, tldraw or variable names/keywords like keyof : https://github.com/100mslive/100ms-docs/pull/1841/checks?check_run_id=18431830227 cc @hdz-666
> 
> 
@akankshat100ms vale is still giving some warning just check by clicking on the action on left, if those warnings are not from your change ignore them
@KaustubhKumar05 let's increase severity for vale.

**Review by saikatmitra91 — APPROVED** [review]

**Comment by akankshat100ms** [comment]
> Demo pic missing in Background Colors and Border Colors section.

this was fixed

**Review by mntrspace — APPROVED** [review]
Doneee!

**Review by pawan-100ms — APPROVED** [review]

**Pull request merged**